string input_msg
geometry_msgs/Pose[] pose_list
---
string output_msg
geometry_msgs/Pose[] pose_list